### ISSUE TITLE
[1.29] Cookie policy issues (PR#1793)

### ISSUE THREAD

**Issue by elena-zh** [cowprotocol/cowswap#1825]
Reported in https://github.com/cowprotocol/cowswap/pull/1793#pullrequestreview-1228952741
1. Fixed in #1821
2. ~~We should use the same format for all 'Last updated' sentences across all content pages~~
@alfetopito [suggestion](https://github.com/cowprotocol/cowswap/pull/1791#issuecomment-1365259375):
```
I would make sure all of them (T&C, Cookie and Privacy policy) follow the same pattern.

For example. 
- This: `(Last Updated: November 2022)`
- Cookie policy: `(updated: November 2022)`
- T&C: `Last updated: November 2022`
```

3. I don't think we need an additional background area for paragraphs. I think, we need to use the same format for all content pages (no additional bg): 
![no need in bg](https://user-images.githubusercontent.com/70885163/209340140-45c1fc86-f95b-4836-af40-b81d8419e5e3.png)
4. Do we really need to highlight it in red? It is not highlighted in the doc:
![not in red](https://user-images.githubusercontent.com/70885163/209340288-02acab1f-5773-4630-bc4f-f3348719b588.png)

5. For me, it is difficult to scroll the table: long press selects and highlights the text, but does not scroll the table content. Can we add a scroll effect on the long press action? 
![image](https://user-images.githubusercontent.com/70885163/209340698-392e4a93-2fdc-417b-9721-8160c228bffc.png)
6. It would be great to enhance the scroll bar itself in Win11
![image](https://user-images.githubusercontent.com/70885163/209340842-4553917f-ed83-491d-9b12-2243d4de8aa7.png)
7. The table in a mobile view does not look good: 
![image](https://user-images.githubusercontent.com/70885163/209340926-8a56a536-1334-4aa0-ac46-d703d7c7e17b.png)
8. Some links exceed the screen width in a mobile view:
![image](https://user-images.githubusercontent.com/70885163/209340997-b734eeee-d6c2-4846-a798-978a967a8b02.png)

**Comment by alfetopito** [comment]
FYI, I've addressed point #2 on https://github.com/cowprotocol/cowswap/pull/1821/files

I just forgot to mention it on the PR description

**Comment by elena-zh** [comment]
> FYI, I've addressed point #2 on https://github.com/cowprotocol/cowswap/pull/1821/files
> 
> I just forgot to mention it on the PR description

Yes, thank you! The only one nitpick now is that `Last updated: November 2022` is in italic on T&C page, and is not in italic on Policies pages. I think, it would be nice to make the style the same everywhere
![image](https://user-images.githubusercontent.com/70885163/212330702-e76ac9a6-173c-4f07-9169-e06dcb1e9f69.png)
![image](https://user-images.githubusercontent.com/70885163/212330725-b1ced248-343a-47e8-9887-b3c3c3c37732.png)

**Comment by alfetopito** [comment]
> > FYI, I've addressed point #2 on https://github.com/cowprotocol/cowswap/pull/1821/files
> > I just forgot to mention it on the PR description
> 
> Yes, thank you! The only one nitpick now is that `Last updated: November 2022` is in italic on T&C page, and is not in italic on Policies pages. I think, it would be nice to make the style the same everywhere ![image](https://user-images.githubusercontent.com/70885163/212330702-e76ac9a6-173c-4f07-9169-e06dcb1e9f69.png) ![image](https://user-images.githubusercontent.com/70885163/212330725-b1ced248-343a-47e8-9887-b3c3c3c37732.png)

https://github.com/cowprotocol/cowswap/pull/1875
